README.md

```diff
@@ -30,14 +30,24 @@ Edit this document to include your answers after each question. Make sure to lea
 
```

Comment:
All answers are correct. I would suggest you come back to these freestyle questions, as these are asked in interviews.